File: Clases_C#/logic/Usuario.cs
namespace TicketsSoporte.logica
{
    public abstract class Usuario
    {
        public string strCodigo { get; set; }
        public string strNombre { get; set; }
        public string strCorreo { get; set; }
        public bool blnActivo { get; set; }

        public Usuario(string strCodigo, string strNombre, string strCorreo)
        {
            this.strCodigo = strCodigo;
            this.strNombre = strNombre;
            this.strCorreo = strCorreo;
            this.blnActivo = true;
        }

        public abstract string obtenerRol();

        public virtual void mostrarInformacion()
        {
            Console.WriteLine($"[{obtenerRol()}] Código: {strCodigo} | Nombre: {strNombre} | Correo: {strCorreo} | Activo: {blnActivo}");
        }
    }
}

File: Clases_Bluej/Sistema_ticketing/Usuario.java
public abstract class Usuario {
    protected String strCodigo;
    protected String strNombre;
    protected String strCorreo;
    protected boolean blnActivo;

    public Usuario(String strCodigo, String strNombre, String strCorreo) {
        this.strCodigo = strCodigo;
        this.strNombre = strNombre;
        this.strCorreo = strCorreo;
        this.blnActivo = true;
    }

    public abstract String obtenerRol();

    public void mostrarInformacion() {
        System.out.println("[" + obtenerRol() + "] Codigo: " + strCodigo + " | Nombre: " + strNombre + " | Correo: " + strCorreo + " | Activo: " + blnActivo);
    }
}